src/Microsoft.DocAsCode.MarkdownLite/JsonRenderer.cs

```diff
@@ -0,0 +1,310 @@
+        {
+            return this.Insert("Image", this.GetSize(), Type.Size);
+        }
+
+        public virtual StringBuffer Render(IMarkdownRenderer renderer, MarkdownStrongInlineToken token, MarkdownInlineContext context)
+        {
+            var childContent = StringBuffer.Empty;
+            foreach (var item in token.Content)
+            {
+                childContent += renderer.Render(item);
+            }
+            return this.Insert("Strong", childContent.ToString(), Type.Child);
+        }
+
+        public virtual StringBuffer Render(IMarkdownRenderer renderer, MarkdownEmInlineToken token, MarkdownInlineContext context)
+        {
+            var childContent = StringBuffer.Empty;
+            foreach (var item in token.Content)
+            {
+                childContent += renderer.Render(item);
+            }
+            return this.Insert("Em", childContent.ToString(), Type.Child);
+        }
+
+        public virtual StringBuffer Render(IMarkdownRenderer renderer, MarkdownCodeInlineToken token, MarkdownInlineContext context)
+        {
+            return this.Insert("InLineCode", this.GetSize(), Type.Size);
+        }
+
+        public virtual StringBuffer Render(IMarkdownRenderer renderer, GfmDelInlineToken token, MarkdownInlineContext context)
+        {
+            var childContent = StringBuffer.Empty;
+            foreach (var item in token.Content)
+            {
+                childContent += renderer.Render(item);
+            }
+            return this.Insert("Del", childContent.ToString(), Type.Child);
+        }
+
+        public virtual StringBuffer Render(IMarkdownRenderer renderer, MarkdownTagInlineToken token, MarkdownInlineContext context)
+        {
+            return this.Insert("Tag", this.GetSize(), Type.Size);
+        }
+
+        public virtual StringBuffer Render(IMarkdownRenderer renderer, MarkdownBrInlineToken token, MarkdownInlineContext context)
+        {
+            return this.Insert("Br", this.GetSize(), Type.Size);
+        }
+
+        public virtual StringBuffer Render(IMarkdownRenderer renderer, MarkdownTextToken token, MarkdownInlineContext context)
+        {
+            return this.Insert("InLineText(" + token.Content.Replace("\n", "\\n") + ")", this.GetSize(), Type.Size);
+        }
+
+        #endregion
+
+        #region Misc
+
+        public virtual StringBuffer Render(IMarkdownRenderer renderer, MarkdownIgnoreToken token, IMarkdownContext context)
+        {
+            return this.Insert("Ignore", this.GetSize(), Type.Size);
+        }
+
+        public virtual StringBuffer Render(IMarkdownRenderer renderer, MarkdownRawToken token, IMarkdownContext context)
+        {
+            switch (token.Rule.Name)
+            {
+                case "Html":
+                {
+                    return this.Insert("Raw(FromHtml)", this.GetSize(), Type.Size);
+                }
+                case "Inline.AutoLink":
```

Comment:
Agree

---
In reply to: [74871144](https://github.com/dotnet/docfx/pull/608#discussion_r74871144) [](ancestors = 74871144)